rai/client.go

```diff
@@ -1304,6 +1312,16 @@ func (c *Client) GetTransactionProblems(id string) ([]interface{}, error) {
 	return result, nil
 }
 
+func (c *Client) CancelTransaction(id string) (interface{}, error) {
```

Comment:
maybe it should just return an error? or a `CancelTransactionResponse` that's an empty struct? idk 😛